service/http/service.go

```diff
@@ -264,7 +265,7 @@ func (s *Service) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 
 	if s.https != nil && r.TLS != nil {
-		w.Header().Add("Strict-Transport-Security", "max-age=63072000; includeSubDomains")
+		w.Header().Add("Strict-Transport-Security", "max-age=31536000; includeSubDomains; preload")
```

Comment:
Thanks for the PR @antonydevanchi. According to the specification, `The max-age must be at least 31536000 seconds (1 year).`, but it already was `63072000` seconds which is more than 1 year. What is the reason for the `max-age` change?

Reply:
@48d90782 i don't remember reason exactly (april 2020, huh), but it was something like «best practice solution» and best practice was not equals RFC (never happened and now again).

Also, you can check dudes like:
— gmail.com
— onedrive.com
— slack.com
— metrika.yandex.ru
— ...

All this hosts using `max-age=31536000` only.

I run psknv.ru with `strict-transport-security: max-age=63072000; includeSubDomains; preload` and nothing bad happens. Website still get all green lights at https://hstspreload.org/?domain=psknv.ru.

Reply:
Ok) It's looking good for me)